testing/tests/gen_ws_tests.py

```diff
@@ -10,7 +10,10 @@
 IF, ELSE_IF, ELSE, END_IF = 0, 1, 2, 3
 
 NL = "\\n"
-dash = lambda ws: [" ", "-"][ws]
+
+
+def dash(ws, sign="-"):
```

Comment:
Let's rename `sign` here according to what the AST has. I'm not sure what `dash()` means here?

Reply:
Renaming it to `sign` would be problematic for the parameters so I instead renamed it `sign_to_use`.

Reply:
No, I meant it should be more like `suppress`/`preserve` and less like `sign`/`minus`/`plus`.

Reply:
The function or the parameter? :laughing: I guess you meant both. So in this case, the parameter will be `suppress`, but what about the function name?

Reply:
Oh, missed that the name was already dash. I guess the function could be called `suppress` and the param could be called `action` or `kind`?

Reply:
Seems weird to have a function called `suppress` if an argument would actually be doing the opposite no?

Reply:
Maybe `handle_ws()`?

Reply:
Let's go with this one then! :)